logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/featureConfig/SyncFeatureConfigsUseCase.kt

```diff
@@ -63,7 +65,8 @@ internal class SyncFeatureConfigsUseCaseImpl(
             // TODO handle other feature flags and after it bump version in [SlowSyncManager.CURRENT_VERSION]
             guestRoomConfigHandler.handle(it.guestRoomLinkModel)
             fileSharingConfigHandler.handle(it.fileSharingModel)
-            mlsConfigHandler.handle(it.mlsModel)
+            mlsConfigHandler.handle(it.mlsModel, true)
```

Comment:
suggestion: could you please specify the variable name for 'true' value here for better readability?

Reply:
Done